.coveragerc

```diff
@@ -0,0 +1,47 @@
+# .coveragerc for edx-platform
+[run]
+data_file = reports/.coverage
+source =
+    cms
+    common/djangoapps
+    common/lib/calc
+    common/lib/capa
+    common/lib/xmodule
+    lms
+    openedx/core/djangoapps
+
+omit =
+    cms/envs/*
+    cms/manage.py
+    cms/djangoapps/*/migrations/*
+    cms/djangoapps/*/features/*
+    lms/envs/*
+    lms/djangoapps/*/migrations/*
+    lms/djangoapps/*/features/*
+    common/djangoapps/terrain/*
+    common/djangoapps/*/migrations/*
+    openedx/core/djangoapps/*/migrations/*
+
+branch = true
```

Comment:
since we never look at the branch number, unless coveralls does something with it I'd say take that out (leave it with the default value for branch, which is False)

http://nedbatchelder.com/code/coverage/config.html